torch/distributed/distributed_c10d.py

```diff
@@ -854,11 +863,31 @@ def _new_process_group_helper(
             _pg_names[pg] = group_name
         else:
             assert backend.upper() in Backend._plugins, (
-                f"unknown c10d backend type {backend.upper()}"
-            )
-            pg = Backend._plugins[backend.upper()](
-                prefix_store, rank, world_size, timeout
+                f"Unknown c10d backend type {backend.upper()}"
             )
+
+            backend_plugin = Backend._plugins[backend.upper()]
+            creator_fn = backend_plugin.creator_fn
+            extended_api = Backend._plugins[backend.upper()].extended_api
+
+            import inspect
+            if pg_options is None:
+                backend_module = inspect.getmodule(creator_fn)
+                if 'Options' not in dict(inspect.getmembers(backend_module, inspect.isclass)):
```

Comment:
Is this a vestigial change? I don't see any other mention of Options anywhere else in this change.